.env.example

```diff
@@ -32,12 +32,12 @@ REDIS_PASSWORD=null
 REDIS_PORT=6379
 
 MAIL_MAILER=smtp
-MAIL_HOST=mailpit
-MAIL_PORT=1025
+MAIL_HOST=smtp.gmail.com
+MAIL_PORT=587
 MAIL_USERNAME=null
 MAIL_PASSWORD=null
-MAIL_ENCRYPTION=null
-MAIL_FROM_ADDRESS="hello@example.com"
+MAIL_ENCRYPTION=tls
+MAIL_FROM_ADDRESS="no-reply@blumilk.pl"
```

Comment:
Why `gmail`? Locally, we are using mailpit.

Reply:
ok, changed back